docs/release-notes.txt

```diff
@@ -6,6 +6,47 @@ Release Notes
 Version 0.12
 ------------
 
+Version 0.12.13
+^^^^^^^^^^^^^^^
+
+**Release Date:** August 6, 2020
+
+**New Features**
+
+- Add the Determined Model Registry. The Model Registry makes it easy to organize trained models by providing versioning and labeling tools. See :ref:`organizing-models` to get started.
+
+- Add a new PyTorch interface to customize your workflows. See the :ref:`migration guide <migration-guide-flexible-primitives>` to upgrade your PyTorch interface. *Deprecated methods will be retained until at least the next minor release.*
+
+  - Support multiple models, optimizers, and LR schedulers. Instantiate those objects and wrap them with :meth:`PyTorchTrialContext.wrap_model`, :meth:`PyTorchTrialContext.wrap_optimizer`, and :meth:`PyTorchTrialContext.wrap_lr_scheduler` in :meth:`PyTorchTrial.__init__`. ``build_model``, ``optimizer``, and ``create_lr_scheduler`` in :class:`PyTorchTrial` are now deprecated.
```

Comment:
Perhaps "Instantiate" → "Model code should instantiate" (or similar); otherwise, it initially sounds like "Instantiate..." is another feature, parallel to "Support...".